### ISSUE TITLE
Compile Error 

### ISSUE THREAD

**Issue by escapeanaemia** [vimeo/VimeoUpload#236]
#### Issue Summary
Cannot compile VimeoUpload 

#### Reproduction Steps
After adding VimeoUpload and build it, It show 65 error😢

#### What I did
- clean project
- clean Xcode/DerivedData Folder
- pod repo update
- update VimeoNetworking as 4.0.0
- update VimeoNetworking as 4.1.0
- build VimeoUpload scheme in Pods.xcodeproj file  (but not working.. build failed)
- build VimeoNetwork scheme in Podfs.xcodeproj file (build failed too)
#### Error & pod file
<img width="560" alt="스크린샷 2019-11-16 오후 5 15 48" src="https://user-images.githubusercontent.com/19788090/68990282-e21db080-0894-11ea-9f41-422d96d9c287.png">
<img width="582" alt="스크린샷 2019-11-16 오후 5 11 20" src="https://user-images.githubusercontent.com/19788090/68990291-08435080-0895-11ea-9b5a-a724be5ea5c0.png">
<img width="580" alt="스크린샷 2019-11-16 오후 5 10 59" src="https://user-images.githubusercontent.com/19788090/68990293-0aa5aa80-0895-11ea-8303-ce221b808d1f.png">
<img width="581" alt="스크린샷 2019-11-16 오후 5 10 38" src="https://user-images.githubusercontent.com/19788090/68990295-0da09b00-0895-11ea-8e55-b433726e9478.png">

**Comment by rpassis** [comment]
Hi @escapeanaemia there is a mismatch between the version of `VimeoUpload` you are pulling and the version of `VimeoNetworking` required for it to work.

So in your case, since you are specifically locking `VimeoNetworking` to version 4.1.0, I suggest you lock your `VimeoUpload` version to `3.0.0` as they are compatible with each other. 

The alternative is to use both versions off the library off the `develop` branch (this is what we do internally) but be aware that the develop branch changes constantly and may introduce API breaking changes.

In short, this is what your Podfile should look like:

```
pod 'VimeoUpload', :git => 'git@github.com:vimeo/VimeoUpload.git', :tag => '3.0.0'
pod 'VimeoNetworking', '~> 4.1.0'
```

Or if consuming both libraries from `develop`

```
pod 'VimeoUpload', :git => 'git@github.com:vimeo/VimeoUpload.git', :branch => 'develop'
pod 'VimeoNetworking', :git => 'git@github.com:vimeo/VimeoNetworking.git', :branch => 'develop'
```

**Comment by rpassis** [comment]
Hi @escapeanaemia there is a mismatch between the version of `VimeoUpload` you are pulling and the version of `VimeoNetworking` required for it to work.

So in your case, since you are specifically locking `VimeoNetworking` to version 4.1.0, I suggest you lock your `VimeoUpload` version to `3.0.0` as they are compatible with each other. 

The alternative is to use both versions of the library off the `develop` branch (this is what we do internally) but be aware that the develop branch changes constantly and may introduce API breaking changes.

In short, this is what your Podfile should look like:

```
pod 'VimeoUpload', :git => 'git@github.com:vimeo/VimeoUpload.git', :tag => '3.1.0'
pod 'VimeoNetworking', '~> 4.1.0'
```

Or if consuming both libraries from `develop`

```
pod 'VimeoUpload', :git => 'git@github.com:vimeo/VimeoUpload.git', :branch => 'develop'
pod 'VimeoNetworking', :git => 'git@github.com:vimeo/VimeoNetworking.git', :branch => 'develop'
```

**Comment by escapeanaemia** [comment]
It works! thank you so much 😊

**Comment by Ant-Adis** [comment]
Hi @rpassis 
I tried your suggestion above but I get this error:

[!] Error installing VimeoUpload
[!] Failed to download 'VimeoUpload': [!] /usr/bin/git clone git@github.com:vimeo/VimeoUpload.git /var/folders/3r/kvk1mkl95lj5mmvc6wqy4q9m0000gn/T/d20200331-62234-1s1b8wg --template= --single-branch --depth 1 --branch 3.1.0

Cloning into '/var/folders/3r/kvk1mkl95lj5mmvc6wqy4q9m0000gn/T/d20200331-62234-1s1b8wg'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

Can you help, thanks
